Adyen/Utilities/Debugging/Assertion.swift

```diff
@@ -17,12 +17,20 @@ public enum AdyenAssertion {
 
     /// :nodoc:
     /// Calls `assertionFailure` when not runing Tests.
-    public static func assert(message: String) {
+    public static func assert(message: @autoclosure () -> String) {
         if CommandLine.arguments.contains("-UITests") {
-            listner?(message)
+            listner?(message())
             return
         }
+        assertionFailure(message())
+    }
 
-        assertionFailure(message)
+    /// :nodoc:
+    /// Calls `assertionFailure` when not runing Tests.
```

Comment:
Description is wrong. `Calls `assertionFailure` when condition not met`

Reply:
fixed

Reply:
oh.. it is actually "Calls assertionFailure when conditions met"